### PR TITLE
SAK-39936 - Improve "includeWebjarLibraryWithVersion"

### PR DESCRIPTION
**Pull request by mpellicer**

### CODE REVIEW

File: library/src/webapp/js/headscripts.js [hunk 1]
```diff
@@ -752,15 +752,23 @@ function includeLatestJQuery(where) {
...
 		if (portal.pageWebjarsPath) webjars = portal.pageWebjarsPath;
 		if (portal.portalCDNQuery) ver = portal.portalCDNQuery;
 	}
-	document.write('\x3Cscript type="text/javascript" src="'+webjars+library+ver+'">'+'\x3C/script>')
+
+	if (library == 'bootstrap-multiselect') {
+		var bootstrapMultiselectVersion = "0.9.15";
+		document.write('\x3Cscript type="text/javascript" src="'+webjars+'bootstrap-multiselect/'+bootstrapMultiselectVersion+'/js/bootstrap-multiselect.js'+ver+'">'+'\x3C/script>');
+		document.write('\x3Clink rel="stylesheet" href="'+webjars+'bootstrap-multiselect/'+bootstrapMultiselectVersion+'/css/bootstrap-multiselect.css'+ver+'"/>');
+		window.console && console.log('Adding bootstrap-multiselect, version '+bootstrapMultiselectVersion);
```
[the remark this answers is not in the document]
Comment (mpellicer) [reply library/src/webapp/js/headscripts.js]: Is not really needed, but if we start using more webjar components, I think it's good to log the components that a page uses.

What would you like to include? Remove the log line ? Add a log line for "others"?

Thanks @ern

[the remark this answers is not in the document]
Comment (ern) [reply library/src/webapp/js/headscripts.js]: OK so I would add one line log line that logs them all, thanks Miguel.

[the remark this answers is not in the document]
Comment (mpellicer) [reply library/src/webapp/js/headscripts.js]: @ern Done, thanks

**Comment by mpellicer** [comment]
Looks like a good plan @jonespm, thanks for the info.

I've raised https://jira.sakaiproject.org/browse/SAK-39964, lets do this!

**Comment by ern** [comment]
FYI I've addressed item 2 css in #5565

**Pull request merged**